app/templating/summary/answer.py

```diff
@@ -1,7 +1,9 @@
 class Answer:
 
-    def __init__(self, answer_schema, answer):
+    def __init__(self, answer_schema, answer, other_value=None):
```

Comment:
Just wondering, rather than `other_value`, perhaps this should be `child_answer_value` so that it fits in better with the new nomenclature.